src/foundry/client-esm/dice/terms/dice.d.mts

```diff
@@ -374,7 +375,7 @@ declare namespace DiceTerm {
     strict: boolean;
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-empty-interface
+  // eslint-disable-next-line @typescript-eslint/no-empty-object-type, @typescript-eslint/no-empty-interface
```

Comment:
I wanted to update ESLint so that no-empty-interface/no-empty-object-type wasn't going to cause problems.

I started the work in https://github.com/League-of-Foundry-Developers/foundry-vtt-types/pull/2769
But that stalled once I realized that I'd have to migrate over to the new lints since the old ones seem to be completely unavailable?